lib/checkunusedvar.h

```diff
@@ -23,6 +23,7 @@
 
 #include "check.h"
 #include "config.h"
+#include "symboldatabase.h"
```

Comment:
You don't need to include here.. I believe you can put a forward declaration for `Variable` below instead. To make compilation faster.